masonry/src/testing/harness.rs

```diff
@@ -472,8 +472,7 @@ impl TestHarness {
     #[track_caller]
     pub fn mouse_move_to(&mut self, id: WidgetId) {
         let widget = self.get_widget(id);
-        let local_widget_center = (widget.ctx().size() / 2.0).to_vec2().to_point();
-        let widget_center = widget.ctx().widget_state.window_transform * local_widget_center;
+        let widget_center = widget.ctx().global_layout_rect().center();
```

Comment:
I don't think this change is correct, in that a transformed widget might not be hit.